content/docs/user-guide/gems/reference/rendering/amd/atom-tressfx.md

```diff
@@ -1,8 +1,8 @@
 ---
 linkTitle: Atom TressFX
 title: Atom TressFX Gem
-description: The Atom TressFX Gem provides realistic hair and fur simulation and rendering in Atom and Open 3D Engine (O3DE).
+description: The Atom TressFX Gem provides realistic hair and fur simulation and rendering in Atom and Open 3D Engine (O3DE).  It enhances the open source TressFX 4.1 by AMD and includes a more advanced handling of lighting using the Marschner lighting model

```

Comment:
Did you intend to only update the `description `metadata? This will only be seen by search engines, and not by readers.